src/test/java/com/znsio/sample/e2e/screen/amazon/CartScreen.java

```diff
@@ -0,0 +1,35 @@
+package com.znsio.sample.e2e.screen.amazon;
+
+import com.znsio.e2e.entities.Platform;
+import com.znsio.e2e.runner.Runner;
+import com.znsio.e2e.tools.Driver;
+import com.znsio.e2e.tools.Visual;
+import com.znsio.sample.e2e.screen.web.amazon.CartScreenWeb;
+import org.apache.commons.lang3.NotImplementedException;
+import org.apache.log4j.Logger;
+
+import static com.znsio.e2e.runner.Runner.fetchDriver;
+import static com.znsio.e2e.runner.Runner.fetchEyes;
+
+public abstract class CartScreen {
+    private static final String SCREEN_NAME = CartScreen.class.getSimpleName();
+    private static final Logger LOGGER = Logger.getLogger(SCREEN_NAME);
+
+    public static CartScreen get() {
+        Driver driver = fetchDriver(Thread.currentThread()
+                .getId());
+        Platform platform = Runner.fetchPlatform(Thread.currentThread()
+                .getId());
+        LOGGER.info(SCREEN_NAME + ": Driver type: " + driver.getType() + ": Platform: " + platform);
+        Visual visually = fetchEyes(Thread.currentThread()
+                .getId());
+
+        switch (platform) {
+            case web:
+                return new CartScreenWeb(driver, visually);
+        }
+        throw new NotImplementedException(SCREEN_NAME + " is not implemented in " + Runner.platform);
+    }
+
+    public abstract String getAddedProductTitleFromShoppingCartScreen(String productTitle);
```

Comment:
Method name can be getProductTitle() as we will be calling it from the cart class then it already conveying the other information